src/Features/ERDDAP/hooks/tabledap.ts

```diff
@@ -149,6 +149,8 @@ export const getDatasetGroup = async (fetchGroup: FetchGroup, startTime?: Date,
   const resultTimeSeries = resultToTimeseries(
     json,
     fetchGroup.datasets.map((ts) => ts.variable),
+    ...fetchGroup.datasets.map((ts) => ts.dataset_public_name),
```

Comment:
Will this be predictable when not all timeseries will have a `.dataset_public_name` that isn't null?

Reply:
I can't wrap my brain around the double negatives hah. Can you say this another way?

Reply:
I see what is happening here now. You're supplying the function with a map of a couple of different things in the same order, and then it extracts and uses the `dataset_public_name` if avaliable.

Reply:
Yes, that's whats going on. It should account for null values in `dataset_public_name`